I1283 and i1324 cleaning creating repairing devfile section (#841)
codenvy/che-docs: src/main/pages/che-7/end-user-guide/assembly_making-a-workspace-portable-using-a-devfile.adoc

---
title: Making a workspace portable using a Devfile
keywords:
tags: []
sidebar: che_7_docs
permalink: che-7/making-a-workspace-portable-using-a-devfile/
folder: che-7/end-user-guide
summary:
---

:parent-context-of-making-a-workspace-portable-using-a-devfile: {context}

:context: making-a-workspace-portable-using-a-devfile
[id='making-a-workspace-portable-using-a-devfile_{context}']
= Making a workspace portable using a devfile

> To transfer a configured Che 7 workspace, create and export the devfile of the workspace and load the devfile on a different host to initialize a new instance of the workspace. For detailed instructions on how to create such a devfile, see below.

Reviewer: ```suggestion
To transfer a configured Che workspace, create and export the devfile of the workspace and load the devfile on a different host to initialize a new instance of the workspace. For detailed instructions on how to create such a devfile, see below.
```
@MichalMaler, use a link to a doc chapter instead of **see below**.

> To transfer a configured Che 7 workspace, you can create and export the devfile of the workspace and load the devfile on a different host to initialize a new instance of the workspace. For detailed instructions to create such a devfile, see below.

Reviewer: ```suggestion
To transfer a configured Che 7 workspace, create and export the devfile of the workspace and load the devfile on a different host to initialize a new instance of the workspace. For detailed instructions on how to to create such a devfile, see below.
```

Reviewer: @MichalMaler, also substitute "see below" for an actual link to the specific section.
